manifests/plugin.pp

```diff
@@ -77,7 +77,7 @@
     $proxy_port  = undef,
 ) {
 
-  include elasticsearch
+  require elasticsearch
```

Comment:
This hasn't changed yet by the looks of it and is still causing the test failure.